packages/py-api/py_api/main.py

```diff
@@ -28,7 +28,7 @@
 def start() -> None:
     run(
         "py_api.main:main_app", host="0.0.0.0", port=6969,
-        reload=True, log_config=get_log_config(),
+        reload=True, log_config=get_log_config(), **SSL_FILES if not IS_OFFLINE else {},
```

Comment:
reloading should also be disabled for deployed instances

Reply:
Done